src/Commands/SelectStatus.ts.txt

```diff
@@ -0,0 +1,120 @@
+// import { Editor, MarkdownView, View } from 'obsidian';
```

Comment:
I see elsewhere that you've introduced a Feature Toggle, and I do like that idea. I agree that it allows working code to be committed to main, to enable testing before it's ready for prime-time.

https://martinfowler.com/articles/feature-toggles.html

Sorry if this comment is premature (I did check that the PR is ready for review) but the usual behaviour of feature toggles is to allow experimental code to be turned on or off by changing the toggle.

As it's all commented out, I'm unsure how we can test it.

Would it be possible for all the UI code to be uncommented out, and then for the code at run-time to only enable the new features if the feature toggle is turned on?

Reply:
Later: I see that the feature toggle is about whether to show a pop-up menu, and this code is about adding a command.

I still would suggest the new code be activated, even if it's not called, so that if there are any future refactorings, like renaming symbols, IDEs would update the inactivated code too, preventing the commented-out code later needing to be manually updated for any changes made in between.

Reply:
Coming in a separate PR. I added the feature capability in late in the cycle so it is skipped for the status change as the refactor is to large. However the UI will be behind the switch as it may be more problematic. All the UI code is just txt files for the moments as I think it through. This means that this PR can go in and the next one will enable that behind the feature flag. 

When a feature is marked as stable we can have the settings automatically show it in the main body as well with a warning if the user has it disabled stating that it is now stable and available for use.